airbyte-api/src/main/openapi/config.yaml

```diff
@@ -182,6 +182,23 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/workspaces/feedback:
```

Comment:
```suggestion
  /v1/workspaces/update_feedback_status:
```

Reply:
Changed to `tag_feedback_status_as_done`